Packs/FeedNVDv2/Integrations/FeedNVDv2/FeedNVDv2.py

```
@@ -0,0 +1,508 @@
+
+            fields["cvsstable"] = metrics
+
+        if cpes:
+            tags, relationships = parse_cpe_command([d['CPE'] for d in cpes], raw_cve.get('id'))
+            if client.feed_tags:
+                tags.append(str(client.feed_tags))
+
+        else:
+            tags = []
+            relationships = []
+
+        fields["tags"] = tags
+        fields["trafficlightprotocol"] = client.tlp_color
+        indicator["relationships"] = [relationship.to_indicator() for relationship in relationships]
+        indicator["type"] = FeedIndicatorType.CVE
+        indicator["rawJSON"] = raw_cve
+        indicator["fields"] = fields
+        indicator["score"] = calculate_dbotscore(fields.get("cvssscore", -1))
+
+        indicators.append(indicator)
+
+    return indicators
+
+
+def calculate_dbotscore(cvss) -> int:
+    """Returns the correct DBot score according to the CVSS Score
+
+    Args:
+        cvss (str): The CVE cvss score
+
+    Returns:
+        int: The Dbot score of the CVE
+    """
+
+    cvss = float(cvss)
+
+    if cvss == -1:
+        return 0
+    elif cvss < 4.0:
+        return 1
+    elif cvss < 7.0:
+        return 2
+    else:
+        return 3
+
+
+def parse_cpe_command(cpes: list[str], cve_id: str) -> tuple[list[str], list[EntityRelationship]]:
+    """
+    Parses a CPE to return the correct tags and relationships needed for the CVE.
+
+    Args:
+        cpe: A list representing a single CPE, see
+        "https://nvlpubs.nist.gov/nistpubs/legacy/ir/nistir7695.pdf" # disable-secrets-detection
+
+    Returns:
+        A tuple consisting of a list of tags and a list of EntityRelationships.
+
+    """
+
+    cpe_parts = {
+        "a": "Application",
+        "o": "Operating-System",
+        "h": "Hardware"
+    }
+
+    vendors = set()
+    products = set()
+    parts = set()
+
+    for cpe in cpes:
+        cpe_split = re.split(r'(?<!\\):', cpe)
+
+        with contextlib.suppress(IndexError):
```

Lets add a comment about why we suppress the error.